app/scripts/modules/openstack/pipeline/stages/bake/bakeStage.html

```diff
@@ -35,13 +35,57 @@ <h4 ng-if="viewState.loading" class="text-center">
         </label>
       </div>
     </stage-config-field>
-
+    <div class="form-group">
+      <div class="col-md-9 col-md-offset-1">
+        <div class="checkbox">
+          <label>
+            <input type="checkbox" ng-model="showAdvancedOptions">
+            <strong>Show Advanced Options</strong>
+          </label>
+        </div>
+      </div>
+    </div>
     <div ng-class="{collapse: showAdvancedOptions !== true, 'collapse.in': showAdvancedOptions === true}">
       <stage-config-field label="Template File Name" help-key="pipeline.config.bake.templateFileName"
                           ng-if="bakeStageCtrl.showTemplateFileName()">
         <input type="text" class="form-control input-sm"
                ng-model="stage.templateFileName"/>
       </stage-config-field>
+      <stage-config-field label="Extended Attributes" help-key="pipeline.config.bake.extendedAttributes"
+                          ng-if="bakeStageCtrl.showExtendedAttributes()">
+        <table class="table table-condensed packed">
+          <thead>
+          <tr>
+            <th style="width:40%">Key</th>
+            <th style="width:60%">Value</th>
+            <th class="text-right">Actions</th>
+          </tr>
+          </thead>
+          <tbody>
+          <tr ng-repeat="(key,value) in stage.extendedAttributes">
+            <td>
+              <strong class="small">{{key}}</strong>
+            </td>
+            <td>
+              <input type="text" required ng-model="stage.extendedAttributes[key]" value="{{value}}"
```

Comment:
I know this is taken from other `bakeStage` templates, but I don't think you need `value="{{value}}"`.